python/pfs/drp/stella/referenceLine.py

```diff
@@ -21,6 +23,26 @@ class ReferenceLineStatus(Bitmask):
     BROAD = 0x10, "Line is broader than normal"
     BAD = 0xFFFF, "Line is bad for any reason"
 
+    @classmethod
+    def getMasks(cls):
+        """Return the corresponding `MaskHelper`
+
+        for representing the `ReferenceLineStatus` as a bitmask in a
+        `pfs.datamodel.PfsFiberArraySet` or similar.
+        """
+        def findBitSet(bitmask: ReferenceLineStatus) -> int:
+            """Return the position of the only single bit set, or zero
```

Comment:
So the assumption here is that only one single bit would be set in the ``ReferenceLineStatus``? Why is that?

Reply:
Following the discussion on slack I finally understand what this code is doing (mapping mask names to bit positions). This is fine with me.